### ISSUE TITLE
JSON input keys with underscores renamed to camel case

### ISSUE THREAD

**Issue by lrowe** [Shopify/javy#126]
With [javy-arm-macos-v0.3.0.gz](https://github.com/Shopify/javy/releases/download/v0.3.0/javy-arm-macos-v0.3.0.gz) release and wasmtime 0.40.1 I am seeing the input json keys with underscores being mangled:

```js
// test.js
function main(input) {
  return JSON.stringify(input);
}
Shopify = { main };
```
```sh
% javy test.js -o test.wasm
warning: unknown name subsection with id 9 at 1397405
% echo '{"hello_world":1}' | wasmtime run test.wasm
"{\"helloWorld\":1}"
```

**Comment by lrowe** [comment]
I guess this must be deliberate as digging deeper it seem to be implemented in commit https://github.com/Shopify/javy/commit/9184e0686efb47e2ba7b9c8fc81fe85c5b22c0fd.

**Comment by saulecabrera** [comment]
Yeah this is deliberate, mainly to adhere to conventional JS standards. We could potentially make this configurable, if needed.
